WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -66,7 +66,7 @@ def psych_sheet_event
 
   def index
     @competition = competition_from_params
-    @registrations = @competition.registrations.accepted.includes(:user, :registration_events)
+    @registrations = @competition.registrations.accepted.includes(:user, :registration_events).order("users.name")
```

Comment:
I don't want to make this simple change complicated, but...  I'd check if this new query performs OK and if it maybe needs an index.